src/store/index.js

```diff
@@ -170,7 +170,7 @@ const store = new Vuex.Store({
       addresses.forEach(address => dispatch('updateBalance', address));
     },
     async updateBalance({ state: { epoch, balances }, commit }, address) {
-      const balance = await epoch.balance(address);
+      const balance = +await epoch.balance(address);
```

Comment:
@edwarddikgale SDK started to return balances as a string, we can lose precision by converting it to Number

Reply:
As I know, all unobvious stuff should be forbidden by strict AirBnb rules for eslint. While code passes this check I'm trying to write it as simple as possible, so unary plus is preferable for me.

If the account balance contains some text instead of digits then something completely wrong happens and the best is to throw an exception. In this case, `null` is better than to pretend that balance is okay.